pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -174,27 +209,93 @@ func (c *Controller) syncGKENetworkParamSet(ctx context.Context, key string) err
 	}
 
 	params := obj.(*networkv1.GKENetworkParamSet)
+	originalParams := params.DeepCopy()
```

Comment:
is just the other way around, this is the one you have to mutate, `params`reference the one in the cache, you can not mutate that

just switch 
```suggestion
        originalParams := obj.(*networkv1.GKENetworkParamSet)
          params  := originalParams.DeepCopy()
```

Reply:
Will change.

I didn't realize the reference to params was long lived and didn't think it mattered.

Reply:
done